test/test_nn.py

```diff
@@ -6067,16 +6067,19 @@ def _gelu_ref(X):
                 return X * stats.norm.cdf(X)
 
             if contiguous:
-                X = torch.rand(n, m, dtype=dtype)
+                X = torch.rand(n, m, dtype=dtype, requires_grad=True)
             else:
-                X = torch.rand(n, m, dtype=dtype)[:, ::2]
+                X = torch.rand(n, m, dtype=dtype, requires_grad=True)[:, ::2]
             res = F.gelu(X)
-            ref = _gelu_ref(X.numpy())
+            ref = _gelu_ref(X.detach().numpy())
             self.assertEqual(res, ref)
+            gradcheck(F.gelu, [X], eps=1e-4, atol=1e-3)
```

Comment:
this looks suspicious. gradchecks happen in double precision, so a tolerance of 1e-3 looks really high and a custom eps is usually not needed. Any idea what is going on? Can you check some sample inputs to inspect.

Reply:
This part just came from testing the numerical stability for gradchecks. Atol has been removed and now it is using default value.